lib/util/merge-confidence/index.ts

```diff
@@ -191,6 +191,12 @@ export async function initMergeConfidence(): Promise<void> {
   return;
 }
 
+export function getApiToken(): string | undefined {
```

Comment:
why this code move?

Reply:
i prefer to have exported function at the top, but fact being that `queryApi` is above this and is un-exported, ill just move it back 😅